src/main/java/com/palehorsestudios/ard/Game.java

```diff
@@ -131,9 +132,11 @@ String Fight(Player player, String option) {
     /**
      * Method to instigate player using their special power.
      */
-    void UsePower(Player player, String option) {
-        System.out.println("use " + option);
+    String UsePower(Player player, String option) {
+        StringBuilder vsb = new StringBuilder();
+        vsb.append("use " + option);
         player.useSpecialPower();
```

Comment:
You should include the result of player.useSpecialPower() in the returned String.